### ISSUE TITLE
Error getting node pricing - AWS GovCloud

### ISSUE THREAD

**Issue by adl2db** [kubecost/cost-model#624]
Noticed that the pricing was off for nodes, then dug into the logs for cost-model and saw there were some errors in getting the node pricing. 

Setup:
* Cloud: AWS GovCloud West
* Kubernetes Distro/Management: EKS
* Instance Types: On-Demand 
* Prometheus: custom
* helm chart: `cost-analyzer-1.70.0`

```
I1210 12:58:11.712174       1 costmodel.go:892] Error getting node pricing. Error: Invalid Pricing Key "us-gov-west-1,g4dn.xlarge,linux"
I1210 12:58:11.712214       1 costmodel.go:892] Error getting node pricing. Error: Invalid Pricing Key "us-gov-west-1,m5.4xlarge,linux"
I1210 12:58:11.712235       1 costmodel.go:892] Error getting node pricing. Error: Invalid Pricing Key "us-gov-west-1,m5.4xlarge,linux"
I1210 12:58:11.712256       1 costmodel.go:892] Error getting node pricing. Error: Invalid Pricing Key "us-gov-west-1,m5.4xlarge,linux"
I1210 12:58:11.712277       1 costmodel.go:892] Error getting node pricing. Error: Invalid Pricing Key "us-gov-west-1,g4dn.xlarge,linux"
```

Pricing sheet successfully loaded: 
```
I1210 12:54:09.655920       1 awsprovider.go:592] starting download of "https://pricing.us-east-1.amazonaws.com/offers/v1.0/aws/AmazonEC2/current/us-gov-west-1/index.json", which is quite large ...
I1210 12:54:11.221794       1 awsprovider.go:724] done loading "https://pricing.us-east-1.amazonaws.com/offers/v1.0/aws/AmazonEC2/current/us-gov-west-1/index.json"
I1210 12:54:11.221818       1 awsprovider.go:853] Finished downloading "https://pricing.us-east-1.amazonaws.com/offers/v1.0/aws/AmazonEC2/current/us-gov-west-1/index.json"
```

Looking at the UI -> Asserts -> Cluster -> Node

Pricing is still being generated, albeit not matching the pricing sheet

data points:

| instance-type | reported | actual |
|---------------|---------------------|-------------------|
| m5.4xlarge    | 0.76566             | 0.968             |
| g4dn.xlarge   | 0.03200             | 0.663             |
| p2.xlarge     | 0.01794             | 1.08              |

Let me know if any more info is needed!

**Comment by dwbrown2** [comment]
Thanks for reporting, @adl2db! Sounds like you installed via helm chart... if so, would you be able to capture and share a [bug report](http://docs.kubecost.com/bug-report)? You can send to team@kubecost.com or share in our Slack directly if that works for you.

**Comment by AjayTripathy** [comment]
Fixed in #651 ! Expected in our release this week.

**Comment by dwbrown2** [comment]
@adl2db just let us know if you have any additional questions here!
